### PR TITLE
add operator info to Kubeflow documentation

### PR DESCRIPTION
**Pull request by adrian555**
This PR creates the documents for using Kubeflow Operator to install and uninstall Kubeflow. Kubeflow Operator was introduced in kfctl since version 0.7.0. This document will help users understand the Kubeflow Operator and adopt it for the deployment and management.

### CODE REVIEW

**Comment by adrian555** [comment]
cc @animeshsingh @Tomcli

File: content/en/docs/operator/install-operator.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+weight = 5
++++
+
+This guide describes how to install the Kubeflow Operator.
+
+There are different ways to install the Kubeflow Operator, choose one of the following:
+
+## 1. Installing the Kubeflow Operator through the OLM console
+
+If the [Operator Lifecycle Manger](https://github.com/operator-framework/operator-lifecycle-manager) (OLM) and the OLM [console](https://github.com/openshift/console) are installed on your cluster, you can find the Kubeflow Operator from the OperatorHub catalog under the Operators as shown on the console.
```
Comment (Tomcli) [inline content/en/docs/operator/install-operator.md:13]:
```suggestion
If the [Operator Lifecycle Manager](https://github.com/operator-framework/operator-lifecycle-manager) (OLM) and the OLM [console](https://github.com/openshift/console) are installed on your cluster, you can find the Kubeflow Operator from the OperatorHub catalog under the Operators as shown on the console.
```

File: content/en/docs/operator/install-operator.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+```shell
+kubectl get pod -n operators
+
+NAME                                 READY   STATUS    RESTARTS   AGE
+kubeflow-operator-55876578df-25mq5   1/1     Running   0          17h
+```
+
+## 2. Installing the Kubeflow Operator through the [operatorhub.io](https://operatorhub.io/operator/kubeflow)
+
+If your cluster does not have OLM installed, navigate to the [operatorhub.io](https://operatorhub.io/operator/kubeflow), click on the `Install` and follow the instructios there to install the operator.
```
Comment (Tomcli) [inline content/en/docs/operator/install-operator.md:30]:
```suggestion
If your cluster does not have OLM installed, navigate to the [operatorhub.io](https://operatorhub.io/operator/kubeflow), click on the `Install` and follow the instructions there to install the operator. 
```

File: content/en/docs/ibm/deploy/install-kubeflow.md [hunk 3]
```diff
@@ -4,8 +4,7 @@ description = "Instructions for deploying Kubeflow with the shell"
 weight = 4
 +++
 
-This guide describes how to use the kfctl binary to
-deploy Kubeflow on IBM Cloud.
+This guide describes how to use the kfctl binary to deploy Kubeflow on IBM Cloud.
```
Comment (animeshsingh) [inline content/en/docs/ibm/deploy/install-kubeflow.md:7]:
```suggestion
This guide describes how to use the kfctl binary to deploy Kubeflow on IBM Cloud Kubernetes Service (IKS).
```

File: content/en/docs/operator/install-kubeflow.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+title = "Installing Kubeflow"
+description = "Instructions for Kubeflow deployment with Kubeflow Operator"
+weight = 10
++++
+
+This guide describes how to use the Kubeflow Operator to deploy Kubeflow.
+
+## Prerequisites
+
+* Follow the [`Install the Kubeflow Operator`](/docs/operator/install-operator) guide to install the Kubeflow Operator
```
Comment (animeshsingh) [inline content/en/docs/operator/install-kubeflow.md:11]:
```suggestion
* Kubeflow Operator needs to be deployed on your cluster for rest of steps to work. Please follow the [`Install the Kubeflow Operator`](/docs/operator/install-operator) guide to install the Kubeflow Operator
```

File: content/en/docs/operator/install-operator.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+weight = 5
++++
+
+This guide describes how to install the Kubeflow Operator.
+
+There are different ways to install the Kubeflow Operator, choose one of the following:
+
+## 1. Installing the Kubeflow Operator through the OLM console
+
+If the [Operator Lifecycle Manager](https://github.com/operator-framework/operator-lifecycle-manager) (OLM) and the OLM [console](https://github.com/openshift/console) are installed on your cluster, you can find the Kubeflow Operator from the OperatorHub catalog under the Operators as shown on the console.
```
Comment (animeshsingh) [inline content/en/docs/operator/install-operator.md:13]:
```suggestion
If the [Operator Lifecycle Manager](https://github.com/operator-framework/operator-lifecycle-manager) (OLM) and the OLM [console](https://github.com/openshift/console) are installed on your cluster, you can find the Kubeflow Operator from the OperatorHub catalog under the Operators as shown on the console. OLM is officially supported on OpenShift
```

File: content/en/docs/operator/install-kubeflow.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
++++
+title = "Installing Kubeflow"
+description = "Instructions for Kubeflow deployment with Kubeflow Operator"
+weight = 10
++++
+
+This guide describes how to use the Kubeflow Operator to deploy Kubeflow.
```
Comment (animeshsingh) [inline content/en/docs/operator/install-kubeflow.md:7]:
```suggestion
This guide describes how to use the Kubeflow Operator to deploy Kubeflow. As mentioned in the Operator [introduction](/docs/operator/introduction.md), the Operator also allows you to monitor and manage the Kubeflow installation beyond the initial installation.
```

File: content/en/docs/operator/install-operator.md (hunk 5 above)
Comment (adrian555) [reply content/en/docs/operator/install-operator.md]: Done

**Pull request merged**